modules/aws/vpn/vpn.tf

```
@@ -16,20 +16,28 @@ resource "aws_vpn_gateway" "vpn_gw" {
   count  = "${var.aws_customer_gateway_id_0 == "" ? 0 : 1}"
   vpc_id = "${var.aws_vpn_vpc_id}"
 
-  tags {
+  tags = {
     Name                         = "${var.aws_vpn_name}"
     "giantswarm.io/installation" = "${var.aws_cluster_name}"
   }
 }
 
+data "aws_vpn_gateway" "vpn_gw" {
+  # Workaround for a bug that claimed to be addressed by 0.12 version.
+  # https://github.com/hashicorp/terraform/issues/12570
+  count = "${var.aws_customer_gateway_id_0 == "" ? 0 : 1}"
+
+  id = "${aws_vpn_gateway.vpn_gw.*.id[count.index]}"
```

What's the thing here with `*` and having `[count.index]` in the end? Could this be like following:
```suggestion
  id = "${aws_vpn_gateway.vpn_gw.[count.index].id}"
```

...below in untouched part there seems to be a second occurance of this.

---

This block should be dropped completely. It is not in use